lib/client/setClientHeader.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+function setClientHeader(method,model,options){
+    options.headers = options.headers || {};
+    options.headers['brisket-client'] = true;
```

Comment:
In my mind, I wanted to restrict it to sync requests like in lib/util/qualifyUrlWithHost.js. I'm not sure adding it to all ajax requests would be the most intuitive.